app/helpers/mailer/send.js

```diff
@@ -1,37 +1,40 @@
 const _ = require('lodash')
 const app = require('@/app')
-const Axios = require('axios')
 const errors = require('@/errors')
+const SES = require('aws-sdk/clients/ses')
 
 module.exports = async function send(emails, sender = {}, templateId) {
-  const personalizations = _.castArray(emails).map(e => ({
-    to: [{ email: e.recipient }],
-    dynamic_template_data: e.body || {}
-  }))
+  const ses = new SES({
+    accessKeyId: app.config.AWS_ACCESS_KEY_ID,
+    secretAccessKey: app.config.AWS_SECRET_ACCESS_KEY,
+    region: app.config.AWS_REGION,
+  })
 
-  const headers = {
-    'Content-Type' : 'application/json',
-    'Authorization' : `Bearer ${app.config.mailer.API_KEY}`
-  }
-
-  const payload = {
-    'personalizations': personalizations,
-    'from': {
-      'email': app.config.mailer.EMAIL,
-      'name':  sender.name || 'UFABC next'
-    },
-    'reply_to': {
-      'email': sender.email || app.config.mailer.EMAIL,
-      'name': sender.name || 'UFABC next'
-    },
-    'template_id': templateId
-  }
+  const personalizations = _.castArray(emails).map((e) =>
+    ses
+      .sendEmail({
+        Source: app.config.mailer.EMAIL,
+        Destination: {
+          ToAddresses: [e.recipient],
+        },
+        Template: templateId,
+        TemplateData:
+          templateId === 'Confirmation'
+            ? `{ \"url\":\"${e.body.url.replace(/\//g, '\\/')}\" }`
```

Comment:
@ViniciussSantos I'd check if this extra escaping is actually necessary, it makes the code terrible to understand, if so might want to create a helper:

```ts
const escapeString = (str) => `\"${str.replace(/\//g, '\\/')}\"`

`{ "url": ${escapeString('https://google.com.br/aa')} }`
```

Reply:
The escaping is necessary according to the SES docs, but the helper is a good choice